### PR TITLE
Script that describes how to migrate traffic bills from observium.

### PR DESCRIPTION
**Pull request by manonfgoo**
This script describes how to migrate traffic bills from observium to librenms.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Cool!
Could you add this to the documentation instead? there is a page for migrating from Observium

**Comment by manonfgoo** [comment]
> 

i put it to the migration folder because it completes the Observium migration. There is no use for it without this

**Comment by Jellyfrog** [comment]
I see. I thought those scripts were old and broken

**Comment by manonfgoo** [comment]
> I see. I thought those scripts were old and broken

No, the scripts are working. AFAIK the XML Conversion is only required when moving from 32- to 64 bit.  The adding of the hosts and RRD file copying worked well for us.

**Comment by PipoCanaja** [comment]
force test run

File: scripts/Migration/Traffic_bills.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+COMMIT ;
+SELECT count(bill_id) from  librenms.bill_data;
+UNLOCK TABLES;
+
+-- Please compare if the  count(bill_id) values are reasonable
+```
+
+### Go to the WEB UI
+
+Now  chek if everything worked OK and reconnect the switchports in the GUI.
```
Comment (laf) [inline scripts/Migration/Traffic_bills.md:77]:
```suggestion
Now  check if everything worked OK and reconnect the switchports in the GUI.
```

**Comment by laf** [comment]
Forcing tests to run

**Comment by manonfgoo** [comment]
Squashed all commits into one

**Comment by manonfgoo** [comment]
> Cool! Could you add this to the documentation instead? there is a page for migrating from Observium

Would you rather put the file in `doc/Installation/` and  Refer to it from `doc/Installation/Migrating-from-Observium.md`?

File: scripts/Migration/Traffic_bills.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+### Precaution
+backup your databases first:
+*  ``mysqldump observium > observium.sql``
+*  ``mysqldump librenms > librenms.sql``
+
+### Warning: traffic bills will be deleted from librenms and imported from observium!
+
+### Initial Import
+
+```
```
[the remark this answers is not in the document]
Comment (manonfgoo) [reply scripts/Migration/Traffic_bills.md]: pls check the commit

**Review by murrant — APPROVED** [review]
Looks good to me pending tests.

**Pull request merged**